frontend/src/lib/components/TaxonomicFilter/InfiniteList.tsx

```diff
@@ -41,19 +45,44 @@ export function tooltipDesiredState(element?: Element | null): ListTooltip {
     return desiredState
 }
 
+const staleIndicator = (parsedLastSeen: dayjs.Dayjs | null): JSX.Element => {
+    return (
+        <Tooltip
+            title={
+                <>
+                    This event was last seen <b>{parsedLastSeen ? parsedLastSeen.fromNow() : 'a while ago'}</b> and
+                    looks like is no longer used.
+                </>
+            }
+        >
+            <Tag className="lemonade-tag">Stale</Tag>
+        </Tooltip>
+    )
+}
+
 const renderItemContents = ({
     item,
     listGroupType,
 }: {
-    item: PropertyDefinition | CohortType
+    item: EventDefinition | CohortType
     listGroupType: TaxonomicFilterGroupType
 }): JSX.Element | string => {
+    const parsedLastSeen = (item as EventDefinition).last_seen_at ? dayjs((item as EventDefinition).last_seen_at) : null
+    const isStale =
+        (listGroupType === TaxonomicFilterGroupType.Events && !parsedLastSeen) ||
```

Comment:
I'm not 100% sure of `!parsedLastSeen`, it means that for a while this stale indicator may throw some false positives, where an event hasn't been received since we push this change, but removing it means that very old events that are no longer sent will never be marked as stale.

Reply:
Perhaps remove the "and looks like is no longer used" part if there's no clear information on its usage? Probably OK to still show "stale" then...